client/ng/components/live-chat/message/message.component.js

```diff
@@ -0,0 +1,22 @@
+// jshint esversion: 6
+
+import module from './message.module';
+import template from './message.html';
+
+class Message {
```

Comment:
Jeżeli komponent nie zawiera żadnej logiki to nie ma sensu zostawiać pustego kontrolera. Usuń klasę, która nic nie robi.